src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -459,6 +413,49 @@ internal void ApplyConflictResolutionEdits(IInlineRenameReplacementInfo conflict
                 }
             }
 
+            private static async Task<IEnumerable<TextChange>> GetTextChangesFromTextDifferencingServiceAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Consider putting this back above `ApplyConflictResolutionEdits` so that the diff is cleaner.